web/wp-content/themes/borealis-2021/src/js/scripts/location.js

```diff
@@ -0,0 +1,99 @@
+function locationModal(locationContainer) {
+    const openVideo = locationContainer.querySelector("#open-vid");
+    const videoModal = locationContainer.querySelector("#video-modal");
+    const openImage = locationContainer.querySelector("#open-img");
+    const imageModal = locationContainer.querySelector("#img-modal");
+    const containerHTML = document.querySelector("html");
+    const imageSlideShow = locationContainer.querySelector("#image-slideshow");
+
+    const classToggle = (el, classToRemove, classToAdd) => {
+        if (classToAdd) {
+            classToAdd.forEach((classname) => {
+                el.classList.add(classname);
+            });
+        }
+        if (classToRemove) {
+            classToRemove.forEach((classname) => {
+                el.classList.remove(classname);
+            });
+        }
+    };
+
+    const getRandomImage = () => {
+        if (imageSlideShow) {
+            const randomNum = parseInt(
+                Math.random() * imageSlideShow.childElementCount
+            );
+            const selectImg = locationContainer.querySelector(
+                `#image${randomNum + 1}`
+            );
+            const images = locationContainer.querySelectorAll(`.image-slideshow`);
+            images.forEach((image) => {
+                classToggle(image, ["opacity-1"], ["opacity-0"]);
```

Comment:
Actually if you're just toggling opacity you can have alt for each image and they'll all always be available for screen reader who can just read the images in order if that works!